doc/radosgw/multisite.rst

```diff
@@ -41,10 +41,10 @@ configurations for the Ceph Object Gateway:
 
 - **Multi-zonegroups:** Ceph Object Gateway supports multiple zonegroups (which
   were formerly called "regions"). Each zonegroup contains one or more zones.
-  If two zones are in the same zonegroup, and if that zonegroup is in the same
-  realm as a second zonegroup, then the objects stored in the two zones share
-  a global object namespace. This global object namespace ensures unique
-  object IDs across zonegroups and zones.
+  **WHAT?** If two zones are in the same zonegroup, and if that zonegroup is in
```

Comment:
@mattbenjamin @yehudasa: I think something's wrong with this sentence. 

This sentence asks us to imagine that two zones are in the same zonegroup, and the zonegroup is in a realm with another zonegroup. The part of the sentence after "then" informs us that the two aforementioned zones share a global namespace. So why does this part of the documentation bother mentioning the second zonegroup?